README.md

```diff
@@ -68,69 +68,73 @@ should be used. Older releases are no longer supported.
-A simplified example: This state develops after steps: create 63 GB namespace, create 1 GB namespace, delete 63 GB namespace. The free capacity is 127 GB, but the driver fails to create a namespace bigger than 64 GB. 
+<sup>1 </sup> **Free space fragmentation** is a problem when there appears to be
+enough free capacity for a new namespace, but there isn't a contiguous region
+big enough to allocate it. The PMEM-CSI driver is only capable of allocating
+continguous memory to a namespace and cannot de-fragment or combine smaller
+blocks. For example, this could happen when you create a 63 GB namespace,
+follwed by a 1 GB namespace, and then delete the 63 GB namespace. Eventhough
+there is 127 GB available, the driver cannot create a namespace larger than
+64 GB. 
 
 ```
 ---------------------------------------------------------------------
 |         63 GB free         | 1GB used |         64 GB free        |
 ---------------------------------------------------------------------
 ```
 
-<sup>2 </sup> *Region affinity* means that all parts of a provisioned file system are physically located on device(s) that belong to same PMEM region. This is important on multi-socket systems where media access time may vary based on where the storage device(s) are physically attached.
+<sup>2 </sup> **Region affinity** means that all parts of a provisioned file
+system are physically located on device(s) that belong to same PMEM region.
+This is important on multi-socket systems where media access time may vary
+based on where the storage device(s) are physically attached.
 
-<sup>3 </sup> *fsdax* mode refers to the modes of NVDIMM
+<sup>3 </sup> **fsdax mode** is required for NVDIMM
 namespaces. See [Persistent Memory
 Programming](https://pmem.io/ndctl/ndctl-create-namespace.html) for
-details. The *devdax* mode is not supported. It might make sense for a
-raw block volume when the application does not need a filesystem. But
-the *devdax* mode creates a character device, something that
-Kubernetes does not expect from a storage driver and cannot handle
-when it [binds the device to a loop
-device](https://github.com/kubernetes/kubernetes/blob/7c87b5fb55ca096c007c8739d4657a5a4e29fb09/pkg/volume/util/util.go#L531-L534),
+details. `devdax` mode is not supported. Though a
+raw block volume would be useful when a filesystem isn't needed, Kubernetes cannot handle [binding a character device to a loop device](https://github.com/kubernetes/kubernetes/blob/7c87b5fb55ca096c007c8739d4657a5a4e29fb09/pkg/volume/util/util.go#L531-L534).
 
 ### LVM device mode
 
-The following diagram illustrates the operation in LVM device mode:
-![devicemode-lvm diagram](/docs/images/devicemodes/pmem-csi-lvm.png)
+In <abbr title="Logical Volume Management">LVM</abbr> mode the PMEM-CSI driver
```

Comment:
What does `<abbr>` do? It doesn't seem to have an effect in the rendered document on GitHub.

Reply:
It makes the meaning of abbreviation or acronym appear when hovering over it (like a tooltip). I'm surprised GitHub appears to be stripping it out. I'll remove it.